Public/Remove-WinADDuplicateObject.ps1

```diff
@@ -24,6 +24,9 @@
     $Count = 0
     $DuplicateObjects = Get-WinADDuplicateObject @getWinADDuplicateObjectSplat
     foreach ($Duplicate in $DuplicateObjects | Select-Object -First $LimitProcessing) {
+        If($Duplicate.ProtectedFromAccidentalDeletion -eq "True"){
```

Comment:
It should be $true - it's not a string, it's a bool as far as I know

Reply:
Modified it to $true